server/config.go

```diff
@@ -56,10 +57,8 @@ type Config struct {
 	// Etcd onlys support seoncds TTL, so here is second too.
 	LeaderLease int64 `toml:"lease" json:"lease"`
 
-	// Log level.
-	LogLevel string `toml:"log-level" json:"log-level"`
```

Comment:
The default behavior is not changed. Only matters when a user changes default log level or specifies log file path in pd.toml .